app/views/site/copyright.html.erb

```diff
@@ -38,21 +38,20 @@
 <% end %>
 
 <% I18n.with_locale @locale do %>
-  <%= tag.div :lang => @locale, :dir => t("html.dir") do %>
+  <%= tag.div :lang => @locale, :dir => t("html.dir"), :class => "richtext" do %>
```

Comment:
This class gives us some nice styling around the example image and block quotes. Otherwise, the `<blockquote>` tag’s default styles have been reset to look just like an ordinary `<p>`, which would be confusing in this case.